org.scala-ide.sdt.core/src/org/scalaide/refactoring/internal/rename/Rename.scala

```diff
@@ -49,7 +50,11 @@ class Rename extends RefactoringHandler {
           new refactoring.FileSelection(source.file, tree, start, end)
         }
 
-        selection map refactoring.prepare flatMap (_.right.toOption) map {
+        val preparation = selection flatMap { selection =>
```

Comment:
I do exactly the opposite, because I want to hide outer definitions to make it impossible to access the wrong one.